Neos.Flow/Classes/Mvc/Controller/SimpleActionController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace Neos\Flow\Mvc\Controller;
+
+use Neos\Flow\Mvc\ActionRequest;
+use Neos\Flow\Mvc\ActionResponse;
+use Neos\Flow\Mvc\Exception\NoSuchActionException;
+use Neos\Flow\SignalSlot\Exception\InvalidSlotException;
+
+/**
+ * Provides most direct access to our request/response abstraction,
+ * the request comes directly from the dispatcher and goes directly back to it.
+ *
+ * Views or other processing needs to be added to your controller as needed.
+ * For helpers to facilitate throws, forwards, redirects and such:
+ * @see SpecialResponsesSupport
+ */
+class SimpleActionController implements ControllerInterface
+{
+    /**
+     * @param ActionRequest $request
+     * @return ActionResponse
+     * @throws NoSuchActionException
+     * @throws InvalidSlotException
+     */
+    public function processRequest(ActionRequest $request): ActionResponse
+    {
+        $request->setDispatched(true);
+        $actionMethodName = $this->resolveActionMethodName($request);
+        return $this->$actionMethodName($request);
```

Comment:
i think we should still ensure that the action is public, maybe at runtime by calling it from another class (maybe via an anonymous class?)

Reply:
Yeah I thought about it, but felt this to be an overprotective precaution, after all it needs to end in `Action` anyways to be even callable, and the signature needs to fit otherwise it will just die as well, and in these simple controllers what else should there be.

Reply:
I think `public` makes more sense, as otherwise code sniffing things will immediately say its unused?

Also it feels like some kind of entry point, even if not a conventional method...

Reply:
Fair point, I 'll change it.